pkg/rancher/rancher.go

```diff
@@ -221,6 +222,8 @@ func (r *Rancher) Start(ctx context.Context) error {
 		}
 	}
 
+	registerForLeaderAndFollower(r.Wrangler)
```

Comment:
You want to register this here: https://github.com/rancher/rancher/blob/b8ced38c3992c74ac365a7291d6424be622bfef9/pkg/controllers/managementapi/controllers.go#L38  This is where we register common indexers for all management nodes.

Reply:
Moved here.